authomatic/adapters.py

```diff
@@ -280,3 +280,36 @@ def set_header(self, key, value):
 
     def set_status(self, status):
         self.response.status = status
+
```

Comment:
Please add a Sphinx doc `autoclass` entry for this adapter in the main docstring (aorund line 30)